### ISSUE TITLE
Add a sponsorships property to Motion

### ISSUE THREAD

**Issue by akuckartz** [popolo-project/popolo-spec#77]
In Germany a motion can be created by a set of persons or a faction or a set of factions. There might be even more possibilities.

BTW: a motion is just one specific type of parliamentary document. There are others. Some work has been put into OpenGovLD (and OParl before that). More work needs to be spent on this to create a common vocabulary. I will prepare a proposal.

---

Add a sponsorships(-like) property to Motion. To review:
- [ ] http://docs.opencivicdata.org/en/latest/proposals/0006.html

**Comment by jpmckinney** [comment]
Can you link to an example where a motion was created by many people, by a faction or by a group of factions?

In most jurisdictions, there is one "primary" creator, and then a number of secondary creators, e.g. co-sponsors in the US.

**Comment by akuckartz** [comment]
A recent one created by a group of factions:

On this page click on the PDF Icon next to "Gem. Antrag nach § 5 (Linke) 16 KB":
http://ratsinformation.stadt-koeln.de/vo0050.asp?__kvonr=48430&voselect=12435

**Comment by jpmckinney** [comment]
Is this a motion? It seems to be more a question by multiple authors to the mayor.

**Comment by akuckartz** [comment]
It is a formal motion to add an item to an agenda (this is just an example, different kinds of motions also exist). In that motion the authors are representing different named factions. The mayor is representing the city.

**Comment by jpmckinney** [comment]
Do you have an example where a motion is created by a faction or set of factions?

JSON-LD has no issue with having a set of people proposing a motion - but JSON Schema would need to be updated.

**Comment by akuckartz** [comment]
In https://github.com/opennorth/popolo-spec/issues/77#issuecomment-67149226 I already provided an example of a motion which was created by a set of factions. It is signed by those persons who can legally sign such documents on behalf of the factions. It is not very relevant who signed the document but very relevant on whose behalf (the factions) they were signed.

**Comment by jpmckinney** [comment]
It's possible to determine which faction they represent based on their memberships at the time of signing. The facts are that they signed, so it's factually people who are signing, not factions.

**Comment by akuckartz** [comment]
Yes, but as I wrote: That Information is _far_ less relevant than the factions supporting the motion. And it is possible that the persons signing the motion will not even be known or encoded. In other words: It is _necessary_ to have a property which can be used to express factions supporting the motion. Otherwise this class can not be used in Germany without extending the set of defined properties.

But as I indicated in https://github.com/opennorth/popolo-spec/issues/77#issue-52028889 : some work has to be done to either merge `Paper` and `Motion` or to make it easy for them to coexist in `opengov`.

**Comment by jpmckinney** [comment]
This seems to be a new requirement. I think `creator` should remain the real, actual creators. We can add a new `sponsors` array, which would actually be useful for a lot of other use cases, and sponsors can either be people or organizations.

**Comment by akuckartz** [comment]
That perhaps makes sense. But the matter regarding `creator` is even more subtle: that someone signs a motion in Germany by no means implies that the same person wrote it (or personally liked to sign it !).

**Comment by jpmckinney** [comment]
I don't think there is a use case for who wrote a motion without signing/proposing it. Motions are certainly drafted by interns, etc. in many legislatures but no one tracks that.

There's the creation of the text of the motion - and then there's the act of making it into a real, proposed motion. "Creator" refers to the second act of creation. The motion doesn't exist in the legislative system until it is proposed (or at least scheduled for proposal via another motion).

I don't think the mental state of the person signing is our concern.

**Comment by akuckartz** [comment]
I am also viewing at this from the perspective of systems operated by administrations. There all sorts of additonal information about drafting processes exist. And at least in Germany such data generally must be transfered to archives in a later stage for future generations.

I will have a further look at the Provenance ontology (PROV). Maybe I find a simple and extendable solution.
